owslib/ogcapi/coverages.py

```diff
@@ -58,6 +58,10 @@ def coverage(self, collection_id: str, **kwargs: dict) -> BinaryIO:
         @param scale_factor: factor by which to scale the resulting coverage
         @type scale_axes: list of tuples
         @param scale_axes: [(axis name, number)]
+        @type datetime: tuple | str
```

Comment:
Can we support a `list` as well?  Would be easier for a user too.

Reply:
Yes. No problem. 

I've also noticed that `subset: [(name, lower bound, upper bound)]` are often provided as `dict[str, tuple[lower bound, upper bound]]` (more similar to typical JSON representation). 

I could add this as well while at it.

Reply:
Sure.